src/host/intercept_dll/CMakeLists.txt

```
@@ -50,6 +50,7 @@ if((CMAKE_CXX_COMPILER_ID MATCHES "Clang") OR (CMAKE_CXX_COMPILER_ID MATCHES "GN
 	set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
 
 	target_link_libraries(${INTERCEPT_EXTENSION_NAME} "libstdc++.a")
+	target_link_libraries(${INTERCEPT_EXTENSION_NAME} "pthread")
```

target_link_libraries (${INTERCEPT_PLUGIN_NAME} ${CMAKE_THREAD_LIBS_INIT})
```suggestion
	find_package (Threads)
	target_link_libraries (${INTERCEPT_EXTENSION_NAME} ${CMAKE_THREAD_LIBS_INIT})
```
the find_package actually belongs on a higher level. But shouldn't hurt.

---

Updated with suggested change